code/src/Components/Artist.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const Artist  = (props) => {
+  return (
+  <>
+     <div className="artistName">
+        {props.album.map(artist => {
```

Comment:
This is where the naming from App.js is getting us confused, when we first didn't get what "album" was doing.